tests/src/kakarot/test_kakarot.py

```diff
@@ -126,10 +126,13 @@ def test_register_account_should_fail_caller_not_resolved_address(
             with cairo_error():
                 cairo_run("test__register_account", evm_address=EVM_ADDRESS)
 
-    @pytest.mark.NoCI
     class TestEthCall:
         @pytest.mark.SolmateERC20
-        async def test_erc20_transfer(self, get_contract):
+        @SyscallHandler.patch(
```

Comment:
I dont see a reason to skip this in CI, but to run we need to mock the valid jumdpests

1. either we conside the bytecode "valid" (optimisticallly assuming Solmate is rightfully compiled) and we can say that each query returns True
2. or we run a real analysis on the bytecode (probably not worth it)

Reply:
If the purpose is to run it with profiling I can keep it NoCI - otherwise I think returning True is reasonable.